test/language-transformer-cycles.test.js

```diff
@@ -108,7 +108,7 @@ describe('Deinflection data', () => {
         const dirname = pathDirname(fileURLToPath(import.meta.url));
 
         /** @type {import('language-transformer').LanguageTransformDescriptor} */
-        const descriptor = parseJson(readFileSync(join(dirname, '../ext/data/language/japanese-transforms.json'), {encoding: 'utf8'}));
+        const descriptor = parseJson(readFileSync(join(dirname, '../ext/js/language/ja/language-transforms.json'), {encoding: 'utf8'}));
```

Comment:
This should be run for all the JSON files.

This is a bit related to my comment of "Instead of a bool, should this perhaps be a path to the JSON file?"